### PR TITLE
Add id to redmine task title

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: redmine_to_kanboard.rb [hunk 1]
```diff
@@ -77,7 +77,7 @@ def color_id
   end
 
   def task_title
-    @redmine_issue.subject
+    "Redmine \##{@redmine_issue.id}: #{@redmine_issue.subject}"
```
Comment (ares) [inline redmine_to_kanboard.rb]: it seems quite long and the information is in the link already, what's the usecase? would it be better to have shorten it to "RM #123"? I can imagine a greasemonkey script that would simplify navigation :-)

**Comment by iNecas** [comment]
The use-case is just for the better identification of the issue. I replaced
redmine with rm.

Greasmonky script is not done yet :)

-- Ivan

File: redmine_to_kanboard.rb (hunk 1 above)
Comment (waldenraines) [reply redmine_to_kanboard.rb]: What about just `#1234` instead of `Redmine #1234` or `RM #1234`?

**Review by waldenraines — COMMENTED** [review]
Tested this out and it worked well for me.  Minor comment in review.

**Comment by iNecas** [comment]
I had that first, but wanted to indicate a difference between this and BZ,
that could also potentially be the number

-- Ivan

**Comment by ares** [comment]
I prefer `RM #1234` over `#1234` too, I guess we could make that configurable if needed. @waldenraines would you prefer configuration option or are we good to merge as is?

**Comment by waldenraines** [comment]
> I prefer RM #1234 over #1234 too, I guess we could make that configurable if needed. @waldenraines would you prefer configuration option or are we good to merge as is?

I'm fine with `RM #1234`, not a big deal to me.

**Comment by ares** [comment]
so @iNecas please change the prefix to RM and we're good to go :-)

**Comment by iNecas** [comment]
Sry, forgot to push yesterday :) Done now

**Comment by ares** [comment]
Thanks @iNecas, merging

**Pull request merged**